MetaMask/metamask-extension
# Adds PulseChain Mainnet to Suggested Networks (Featured RPCs)

## Explanation
This pull request adds support for PulseChain as a suggested network to MetaMask. MetaMask supports a few other Ethereum-based networks. With the recent launch of PulseChain and the high interest of many users around it, it's important to support this new network and make it more accessible to users.

The solution that this change offers is a new option in MetaMask's add new network dialog called "PulseChain Mainnet". When selected, it will add  the PulseChain network with the correct settings. Users will still be able to switch to other networks as they wish.

PulseChain Settings can be found here [pulsechain.com](https://pulsechain.com)

## Screenshots/Screencaps
<img width="981" alt="Screenshot 2023-05-14 at 14 48 52" src="https://github.com/MetaMask/metamask-extension/assets/24633381/b7d67b54-8a9c-4eb9-83ba-007eb0c7c05b">
<img width="981" alt="Screenshot 2023-05-14 at 14 44 57" src="https://github.com/MetaMask/metamask-extension/assets/24633381/8ead2a89-bfef-44db-b804-d8c8a2bd63ee">
<img width="918" alt="Screenshot 2023-05-14 at 14 18 48" src="https://github.com/MetaMask/metamask-extension/assets/24633381/f09d1d6b-9b86-41f9-8255-6eee78cb8e0c">

## Manual Testing Steps

To manually test this change, reviewers can follow these steps:

- Buid MetMask and Install a local version in the browser
- Open the add-network dialog in MetaMask and verify that "PulseChain" is listed as an option.
- Select "PulseChain" and verify that the UI updates to reflect the new network.
- Send a transaction on the PulseChain network and verify that it goes through successfully.

## Pre-merge author checklist


## Pre-merge reviewer checklist
